seed/models/filter_group.py

```diff
@@ -21,12 +24,47 @@ class FilterGroup(models.Model):
     or_labels = models.ManyToManyField(StatusLabel, related_name="or_filter_groups")
     exclude_labels = models.ManyToManyField(StatusLabel, related_name="exclude_filter_groups")
 
-    def filtered_property_views(self, columns=[], views=None):
-        qd = QueryDict(mutable=True)
-        qd.update(self.query_dict)
-        filters, _annotations, _order_by = build_view_filters_and_sorts(qd, columns, "property")
-        filtered_views = views.filter(filters)
+    def views(self, views, columns=[]):
+        if VIEW_LIST_INVENTORY_TYPE[self.inventory_type][1] == "Property":
+            return self._filtered_property_views(views, columns)
+        elif VIEW_LIST_INVENTORY_TYPE[self.inventory_type][1] == "Tax Lot":
+            return self._filtered_taxlot_views(views, columns)
```

Comment:
It's not strictly needed, just an fyi, if you want to only have one function, you could do something like [this](https://github.com/SEED-platform/seed/blob/Fix-update-display-name/seed/utils/match.py#L257), where the class is variable

Reply:
Oh, perfect, thank you for the pointer.  I'll implement that and collapse these down into a single function.